### PR TITLE
(MAINT) Notice for missing tag v1.0.0

### PR DESCRIPTION
**Pull request by Ramesh7**
## Summary
Provide a detailed description of all the changes present in this pull request.

## Additional Context
Add any additional context about the problem here. 
- [ ] Root cause and the steps to reproduce. (If applicable)
- [ ] Thought process behind the implementation.

## Related Issues (if any)
Mention any related issues or pull requests.

## Checklist
- [ ] 🟢 Spec tests.
- [ ] 🟢 Acceptance tests.
- [ ] Manually verified. (For example `puppet apply`)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 [![Donated by Ranjit](https://img.shields.io/badge/donated%20by-@WhatsARanjit-fb7047.svg)](#transfer-notice)
 
+We would like to inform you that the tag v1.0.0 was mistakenly created on the HEAD commit, but the module was released with an earlier commit. To address this, we have released a new version, v1.0.1.
```
Comment (bastelfreak) [inline README.md]: maybe also mention that the git tag was deleted again?

File: README.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 [![Donated by Ranjit](https://img.shields.io/badge/donated%20by-@WhatsARanjit-fb7047.svg)](#transfer-notice)
 
+We would like to inform you that during the release of version 1.0.1, the corresponding tag v1.0.0 was not created. To address this, we are publishing a new release, v1.0.1.
```
Comment (bastelfreak) [inline README.md]: The statement is not correct. the v1.0.0 tag was created and then deleted again.

Comment (Ramesh7) [reply README.md]: updated

File: README.md (hunk 1 above)
Comment (Ramesh7) [reply README.md]: resolving as we have updated with new content.

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**